src/Web/HTML.js

```diff
@@ -1,3 +1,3 @@
-export function window() {
+export function windowImpl() {
   return window;
 }
```

Comment:
Can't this fix just be
```js
const windowImpl = function() {
  return window;
}
export windowImpl as window;
```
similar to what we've done for `new`?

Reply:
Done